Rest/Controller/ContentController.php

```diff
@@ -73,13 +75,31 @@ public function __construct(
      *
      * @return \EzSystems\RecommendationBundle\Rest\Values\ContentData
      *
-     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundExceptionif the content, version with the given id and languages or content type does not exist
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException if the content, version with the given id and languages or content type does not exist
      * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException If the user has no access to read content and in case of un-published content: read versions
      */
     public function getContent($contentIdList)
     {
         $contentIds = explode(',', $contentIdList);
-        $content = $this->prepareContent($contentIds);
+        $lang = $this->request->get('lang');
+
+        $criterion = $contentTypesCriterion = array_map(function ($contentId) {
+            return new Criterion\ContentId($contentId);
```

Comment:
This, as most criteria, accepts an array which behaves as an IN in this case:
https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/API/Repository/Values/Content/Query/Criterion/ContentId.php#L44

Besides if you really needed to create a criterion for them all you should have wrapped them in a OR, otherwise no results ;)